src/Pages/Repositories/ContentListTable/components/UploadContent/components/FileUploader.tsx

```diff
@@ -68,15 +77,21 @@ export default function FileUploader({ setFileUUIDs, isLoading }: Props) {
     }
   }, [completedCount, fileCount]);
 
-  const statusIcon = useMemo(() => {
-    if (failedCount) {
-      return <StatusIcon status='failed' removeText />;
-    }
-    if (completedCount === fileCount) {
-      return <StatusIcon status='completed' removeText />;
-    }
-    return <StatusIcon status='running' removeText />;
-  }, [completedCount, fileCount, failedCount]);
+  const isLoadingForParent = useDebounce(isBatching || isHashing || hasRetrying, 200);
+
+  useEffect(() => {
+    setChildLoading(isLoadingForParent);
+  }, [isLoadingForParent]);
+
+  //   const statusIcon = useMemo(() => {
```

Comment:
oh i missed this, was this meant to be commented out?

Reply:
yeah I can safely remove this